### DISCUSSION TITLE
add mongoose.ws

### DISCUSSION THREAD

**Discussion by dvtate** [TechEmpower/FrameworkBenchmarks#9046] [Ideas]
[Mongoose.ws](http://mongoose.ws) has 5x more github stars and seems more mature than a lot of the other libraries in the list. I think it would be good to include it.

**Comment by dvtate** [comment]
also please add Wt: https://www.webtoolkit.eu/wt

**Comment by joanhey** [comment]
You are welcome to add these frameworks, as this benchmark is open source.
